### DISCUSSION TITLE
Application for github student benefit is not submitting?

### DISCUSSION THREAD

**Discussion by fantreatment** [community/community#101922] [GitHub Education] [answered]
### Hi everyone,

Hi, I'm encountering an issue with the GitHub Student Pack. After completing all the required steps, when I click the 'Process Information' button, it gets stuck on 'Processing Information', the last time I tried it, it was like that for a couple hours before I gave up on it. Can you please investigate and provide assistance to resolve this problem?

![Screenshot 2024-01-25 125131](https://github.com/community/community/assets/111869627/6b31f1f4-ab0a-4dff-99ab-f3ee34a5af19)

**Comment by amankr1469** [comment]
Try clearing your browser cache and cookies, then attempt the process again. Or switch to different browser.

**Comment by chalo2000** [comment, marked as answer]
Had this same issue. If you look at the browser console you may see an error there. I had an issue trying to submit my transcript and was getting a 413 Payload too Large POST error. You should try to resize your screenshots if you submitted any.
